endpoints/openrtb2/auction.go

```diff
@@ -138,18 +139,19 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 	}
 
 	usersyncs := usersync.ParsePBSCookieFromRequest(r, &(deps.cfg.HostCookie))
+	setEffectivePubID(req, false, r.URL.Query())
 	if req.App != nil {
 		labels.Source = pbsmetrics.DemandApp
 		labels.RType = pbsmetrics.ReqTypeORTB2App
-		labels.PubID = effectivePubID(req.App.Publisher)
+		labels.PubID = req.App.Publisher.ID
```

Comment:
This still eliminates using the `ParentID` as our label value in metrics? This will also break being able to set our white/blacklists on accounts using the `ParentID` rather than the `Publisher.ID`.